### ISSUE TITLE
Remove models / Re download

### ISSUE THREAD

**Issue by chrsBlank** [nextcloud/llm#11]
Is there a way to remove the model or even re download it?

1) i download llama 2, didnt work

2) i tried downloading gpt4all but download crashed due to space ( i had my temp backup still in my main drive ) and now it straight up crashes even tho it says that the model is downloaded 

i would like to remove the previous model (and the "broken" new one ) and try downloading again but i cant seem to find where it is (manually remove) or a command to remove it

**Comment by Stifler6996** [comment]
Delete them in /var/www/nextcloud/apps/llm/sys-py/models. (Start wherever your default install is)

**Comment by chrsBlank** [comment]
Hi, thought so too.

I get the following /home/hub/nextcloud/apps/llm# cd sys-py
bash: cd: sys-py: No such file or directory

could i look into somewhere else

**Comment by Stifler6996** [comment]
This is the only place. I don't know your system, and it may be split into various install locations (I don't use Docker, so I cannot help you if this is the case).

When you find the base install (or sub-install "base" for the container) it will be ....../apps/llm/sys-py/models

If you know the location of the ./occ file for maintenance etc, then you can probably locate the config and check it. or it might even be in that base directory.

**Comment by marcelklehr** [comment]
@Stifler6996 you have a typo, it's called `src-py` not `sys-py` ;)

**Comment by marcelklehr** [comment]
It should also be possible to force download the model using `download-model -f `

**Comment by Stifler6996** [comment]
> @Stifler6996 you have a typo, it's called `src-py` not `sys-py` ;)

Thanks, I fixed it (sys on the brain!)

I started the download and then cancelled it, so I could check the size before starting it again. Once I knew it was 13GB and I had the space available, I tried to start it again, and it failed.

I then tried to use `download-model -f ` but that failed to work, hence why I know were the partial file was and deleted it manually. Then the normal command worked again.
